app/main.ipynb

```diff
@@ -2,17 +2,21 @@
  "cells": [
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 18,
```

Comment:
Cannot check because of this issue. Try it https://stackoverflow.com/questions/34085676/ipython-notebook-error-notebook-json-is-invalid-uexecution-count-is-a-requi
<img width="529" alt="image" src="https://github.com/mate-academy/py-restaurant-data-analysis/assets/61380547/2c162831-fae2-4eed-839f-ce6671a3cf3b">